pull-weird.js

```diff
@@ -46,10 +43,11 @@ module.exports = function (weird, _done) {
   }
 
   return {
-    source: function (abort, cb) {
+    source (abort, cb) {
```

Comment:
Not only is this pattern shorter, it makes sure that the function is not anonymous. `this` support is preserved

Reply:
Yeah I was looking at that and thinking, ahh that is actually a great usage of the `{ a = a` pattern. Good to know about this.